settings.gradle

```diff
@@ -18,3 +18,11 @@ rootProject.children.each { project ->
 }
 
 enableFeaturePreview('STABLE_PUBLISHING')
+
+buildCache {
+    local(DirectoryBuildCache) {
+        enabled = !System.getenv().containsKey("CI")
+        directory = new File(rootDir, 'build-cache')
```

Comment:
Why do we need this setting or the next one? Can we rely on Gradle's defaults?

Reply:
good question, let's rely on defaults for now and adjust them if needed later on.